core/fast_string.rs

```diff
@@ -471,9 +451,6 @@ macro_rules! __op_name_fast {
 mod tests {
   use super::*;
 
-  // TODO(mmastrac): we need to upstream as_str() on OneByteConst to rusty_v8 to get
-  // rid of the hacks that break miri.
-  #[cfg(not(miri))]
```

Comment:
Have we actually upstreamed this change and just waiting on `rusty_v8` release?

Reply:
It's been out for a bit -- I just forgot about this cleanup.